searx/engines/duden.py

```diff
@@ -16,6 +16,7 @@
     "use_official_api": False,
     "require_api_key": False,
     "results": 'HTML',
+    "language": 'de',
```

Comment:
Should we better add `languages` (plural) with a python list of languages .. if the engine supports several engines (more than three .. or five?) we can use an entry like `languages = ['i18n', ]`  .. suggestions?

Reply:
My idea was that `about.language` specifies the user interface language for engines without multilanguage support (i.e. no `supported_languages`). I'd like to avoid duplicating `supported_languages`.

Reply:
FYI we have a `supported_languages` field for engines, this could be used 

```
$ ./manage pyenv.cmd python
Python 3.8.10 (default, Nov 26 2021, 20:14:08) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import searx.engines
>>> import searx.plugins
>>> searx.engines.load_engines(searx.settings['engines'])
...
>>> searx.engines.engines['google'].supported_languages
...
>>> searx.engines.engines['mediathekviewweb'].supported_languages
```

Sometimes this field is a dict, sometimes it is a list (I don't know why / sorry, I had no time to dig deeper in this topic).

Reply:
Ups, your comment shows up after I send mine / sorry.

> I'd like to avoid duplicating `supported_languages`.

That's fine I think / OK, lets stay with `languages`.

Reply:
To clarify: `supported_languages` can be specified with the `language` parameter.

`about.language` on the other hand is explicitly for engines that don't support multiple languages (and the language thus cannot be specified with the `language` parameter).

Reply:
Thanks!